zerver/lib/upload.py

```diff
@@ -190,10 +199,17 @@ def upload_avatar_image_local(user_file, user_profile, email):
 if settings.LOCAL_UPLOADS_DIR is not None:
     upload_message_image = upload_message_image_local
     upload_avatar_image  = upload_avatar_image_local
+
 else:
     upload_message_image = upload_message_image_s3
     upload_avatar_image  = upload_avatar_image_s3
 
+
+def claim_attachment(path_id, message):
+    #update attachment entry in DB
+    Attachment.objects.filter(path_id=path_id).update(message=message, is_claimed=True)
+    return True
```

Comment:
This should check if it can find the attachment object and provide some sort of reasonable error if it can't find it.